### PR TITLE
Small update bug template

### PR DESCRIPTION
**Pull request by eppingere**
# Small update bug template

## Description
Minor update to bug template since I'm now jenkins minder

## Remaining tasks

None? Hopefully

## Performance
NA

## Further work
???

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/noisepage-issue-template.md [hunk 1]
```diff
@@ -38,10 +38,14 @@ Please provide a short summary of the bug observed.
 ## Environment
 To address the bug, especially if it environment specific, we need to know what kind of configuration you are running on. Please include the following:
 
-**OS**: Ubuntu 18.04 (LTS) or macOS 10.14+ (specify version).
+**OS**: Ubuntu (LTS) 18.04-20.04 or macOS 10.14+ (please specify version).
```
Comment (mbutrovich) [inline .github/ISSUE_TEMPLATE/noisepage-issue-template.md:41]: We only support 20.04 now.

File: .github/ISSUE_TEMPLATE/noisepage-issue-template.md [hunk 2]
```diff
@@ -38,10 +38,14 @@ Please provide a short summary of the bug observed.
...
 
 **Compiler**: GCC 7.0+ or Clang 8.0+. 
 
+**CMake Profile**: Debug/Release or maybe even FastDebug/Relwithdebinfo. If exists across all cmake profiles on a platform-compiler combo, then please say so.
```
Comment (thepinetree) [inline .github/ISSUE_TEMPLATE/noisepage-issue-template.md:45]: Spelling/wording: 

Debug, Release, FastDebug, or RelWithDebInfo. If the issue exists across all CMake profiles on an OS-compiler combo, please mention.

**Review by thepinetree — COMMENTED** [review]
Minor wording nit

File: .github/ISSUE_TEMPLATE/noisepage-issue-template.md (hunk 2 above)
Comment (lmwnshn) [reply .github/ISSUE_TEMPLATE/noisepage-issue-template.md:45]: +1, maybe also format as `Debug`, `Release`, ...

File: .github/ISSUE_TEMPLATE/noisepage-issue-template.md (hunk 1 above)
Comment (eppingere) [reply .github/ISSUE_TEMPLATE/noisepage-issue-template.md:41]: done

File: .github/ISSUE_TEMPLATE/noisepage-issue-template.md (hunk 2 above)
Comment (eppingere) [reply .github/ISSUE_TEMPLATE/noisepage-issue-template.md:45]: done

**Review by thepinetree — APPROVED** [review]
LGTM

**Pull request merged**